apps/saved_searches/saved_searches.py

```diff
@@ -102,8 +102,9 @@ def on_fetched(self, docs):
 class SavedSearchesService(BaseService):
     def on_create(self, docs):
         for doc in docs:
-            if 'user' not in doc and request:
-                doc['user'] = request.view_args.get('user')
+            if 'user' not in doc:
```

Comment:
Maybe 'user' could be removed from the resource as we can infer the user when creating a saved search for the request. Also when update and delete we wouldn't need the user. In this case we can save the user field before committing the changes to database.